example/etcd-backup-operator/backup_cr.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: "etcd.database.coreos.com/v1beta2"
 kind: "EtcdBackup"
 metadata:
-  name: example-etcd-cluster
+  name: example-etcd-cluster-backup
```

Comment:
if we change name here, we also need to change the readme for backup operator.
`kubectl get EtcdBackup example-etcd-cluster -o yaml` => `kubectl get EtcdBackup example-etcd-cluster-backup -o yaml`
https://github.com/coreos/etcd-operator/tree/master/cmd/backup-operator#verify-status

Reply:
Yeah I had the README change in my other branch. Reverting this now. Will do it separately.

Reply:
sounds good!